librosa/feature/spectral.py

```diff
@@ -1586,6 +1586,14 @@ def mfcc(y=None, sr=22050, S=None, n_mfcc=20, dct_type=2, norm='ortho', **kwargs
 
         Normalization is not supported for `dct_type=1`.
 
+    lifter : number >= 0
+        If `lifter>0`, apply *liftering* (cepstral filtering) to the MFCCs:
+
+        `M[n, :] <- M[n, :] * (1 + sin(pi * (n + 1) / lifter)) * lifter / 2`
+
+        Setting `lifter >= 2 * n_mfcc` emphasizes the higher-order coefficients.
```

Comment:
I would love to see a recommended range of values. I have a gut feeling that `lifter=1`  (it cancels one every other MFCC).
"of the order of `n_mfcc`"?

Perhaps the `lifter` should automatically scale with `n_mfcc`? or is that not explicit enough?

Reply:
I thought about normalizing the liftering coefficient by `n_mfcc`, but decided against it for a more transparent relation to prior implementations.  Maybe that's important?